.github/ISSUE_TEMPLATE/first_rc_checklist.md

```diff
@@ -34,7 +34,7 @@ labels: task
 ### Post Release:
 
 * [ ] Clean up `ci_artifacts` by moving files to sub-directories
-* [ ] Tag X.Y+1.0dev0 to start new development cycle on `main`.
+* [ ] Tag `X.Y+1.0dev0` to start new development cycle on `main`.
```

Comment:
Fix the `Annotated tag X.Y.Zrc1` line above too?

Reply:
yes, added in c622c0b774ecf0c63cf4eec9176924e02d52bb50